### PR TITLE
FIX (FTI) model converter for lightGBM models

### PR DESCRIPTION
**Pull request by razdoburdin**
## Description

Fix failure of FTI-model convertor. Performance or quality metric change isn't expected, since only the converter is changed. The CI failures are not related to the PR content.

---
**PR completeness and readability**

- [x] I have reviewed my changes thoroughly before submitting this pull request.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have updated the documentation to reflect the changes or created a separate PR with update and provided its number in the description, if necessary.
- [x] Git commit message contains an appropriate signed-off-by string _(see [CONTRIBUTING.md](https://github.com/uxlfoundation/scikit-learn-intelex/blob/main/CONTRIBUTING.md#pull-requests) for details)_.
- [x] I have added a respective label(s) to PR if I have a permission for that.
- [x] I have resolved any merge conflicts that might occur with the base branch.

**Testing**

- [x] I have run it locally and tested the changes extensively.
- [x] All CI jobs are green or I have provided justification why they aren't.
- [x] I have extended testing suite if new functionality was introduced in this PR.

**Performance**

- [x] I have measured performance for affected algorithms using [scikit-learn_bench](https://github.com/IntelPython/scikit-learn_bench) and provided at least summary table with measured data, if performance change is expected.
- [x] I have provided justification why performance has changed or why changes are not expected.
- [x] I have provided justification why quality metrics have changed or why changes are not expected.
- [x] I have extended benchmarking suite and provided corresponding scikit-learn_bench PR if new measurable functionality was introduced in this PR.

### CODE REVIEW

File: src/gbt_convertors.pyx [hunk 1]
```diff
@@ -180,8 +180,17 @@ class Node:
...
         is_leaf = "leaf_value" in tree
+        if is_leaf:
+            if "leaf_count" in tree:
+                cover=tree["leaf_count"]
+            else:
+                cover=0
+        else:
+            cover=tree["internal_count"]
+
+        value = {tree["leaf_value"] if is_leaf else tree["threshold"]}
```
Comment (ahuber21) [inline src/gbt_convertors.pyx:191]:
```suggestion
        # get cover and value for leaf nodes or internal nodes
        cover = tree.get("leaf_count", 0) or tree.get("internal_count", 0)
        value = tree.get("leaf_value", 0) or tree.get("threshold", 0)
```

I would just write it a bit more compact. Should work for `.pyx`, no?

Comment (razdoburdin) [reply src/gbt_convertors.pyx]: I have just checked this version. It works fine

**Comment by icfaust** [comment]
@razdoburdin I trust @ahuber21 on this one (don't know enough on it tbh)

**Comment by razdoburdin** [comment]
> @razdoburdin I trust @ahuber21 on this one (don't know enough on it tbh)

ok. I will merge it as far as tests will pass
